### DISCUSSION TITLE
How to install Kubernetes on Ubuntu Server?

### DISCUSSION THREAD

**Discussion by heyLeiloo** [heyvaldemar/ama#3] [Q&A] [answered]
Hi! Do you have a guide on how to install Kubernetes?

Thanks!

**Comment by heyvaldemar** [comment, marked as answer]
hey, sure!

[Install Kubernetes on Ubuntu Server 22.04 LTS
](https://www.heyvaldemar.com/install-kubernetes-on-ubuntu-server-22-04-lts/)
